stages/org.osbuild.firewall

```diff
@@ -92,6 +116,15 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_sources in sources:
+        zone = zone_sources["zone"]
+        if len(zone_sources["sources"]) != 0:
```

Comment:
Could have a minimum items constraint on the array and thus avoid the if. Seems like it makes not a lot of sense to have a source with a zone but no sources since the command does not execute then.

Reply:
yep I can make it do that indeed thanks!

Reply:
done (I hope)